airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -764,6 +764,8 @@ def record_process_components(self, record: MutableMapping[str, Any]) -> Iterabl
         # convert dates from ISO-8601 to RFC-3339
         record["published_at"] = self.tools.from_iso8601_to_rfc3339(record, "published_at")
         record["updatedAt"] = self.tools.from_iso8601_to_rfc3339(record, "updatedAt")
+        # unnest `product_count` to the root lvl
+        record["products_count"] = record.get("productsCount", {}).get("products_count")
```

Comment:
[This issue](https://github.com/airbytehq/airbyte-internal-issues/issues/7658) mentions `precision` as well. Should we deal with it here?

Reply:
No, I commented, why not: https://github.com/airbytehq/airbyte-internal-issues/issues/7658#issuecomment-2126951163